### DISCUSSION TITLE
Checking out cdtcloud-blueprint

### DISCUSSION THREAD

**Discussion by Anitha-y** [eclipse-cdt-cloud/cdt-cloud#10] [General]
I am a regular user of Eclipse CDT and was checking out moving to Eclipse Theia for a web based development environment. When i came across this cdtcloud-blueprint I got pretty excited and installed the application on my windows platform. I also referred to an EclipseCon talk about this. In that video they referred to an example cpp workspace that they mentioned would come with the product. However I was not able to find the same both in the installation nor in the Github repository. Where can I find this and how do I go about playing around with cdtcloud. 

Thanks.

**Comment by sdirix** [comment]
Hi @Anitha-y, thanks for your interest in CDT.cloud Blueprint.

The CDT.cloud Blueprint is currently transferred to its [own cdt-cloud-blueprint repository](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint) in the new CDT.cloud project. Once the transfer is complete you can find a detailed description including the example workspace there.

Until then the CDT.cloud Blueprint can be found on [this branch](https://github.com/eclipse-theia/theia-blueprint/tree/jf/wip/c-extend) in the Theia Blueprint repository and the example workspace used in the EclipseCon talk can be found [here](https://github.com/eclipsesource/cdtcloud-alpha/tree/master/example/workspace).

**Reply by Anitha-y** [reply]
Thanks, I will try it out.

**Reply by Anitha-y** [reply]
The example workspace did not have all the files as mentioned in the talk like the compile_commands.json file.

**Reply by sdirix** [reply]
The "compile_commands.json" file is automatically generated via the CMake build. You can execute `cd cpp && cmake .` within the workspace to execute the CMake build or use the `Binary build` task which is part of the workspace.
